test/transformers/test_jsd.py

```diff
@@ -5,6 +5,7 @@
 import torch
```

Comment:
revert the test. i will do an overhaul for the tests later

Reply:
Reverted the `test_jsd.py` and `test_kl_div.py`